### ISSUE TITLE
Links in post headers have newline characters

### ISSUE THREAD

**Issue by lexming** [sunpy/ablog#177]
### Describe the bug

[PR#112](https://github.com/sunpy/ablog/pull/112) changed how links in headers of posts are handled and as a result the `href` element now has newline characters. This breaks our parsers for those html files.

### To Reproduce

All post headers in the post feed pages such as `posts/index.html` are affected.

### Screenshots

```
  <div class="section ablog-post">
    <h2 class="ablog-post-title">
      <a href="
../../../news/2020/electricity-works-cooling-system/
">Electricity works on the cooling system</a>
    </h2>

    <ul class="ablog-archive">
      <li>
         <i class="fa fa-calendar"></i> 02/06/2020
      </li>

<li id="author">
  <span
    ><i class="fa-fw fa fa-user"></i></span
  >
```

### System Details

- ablog version >= 0.10.31

### Installation method

pip

**Comment by nabobalis** [comment]
Thanks for the report, I'll debug and get a fix out.

**Comment by nabobalis** [comment]
I released v0.10.33, can you check it works for you?

Limited checks on RTD shows that the line breaks are gone now.

**Comment by lexming** [comment]
Version 0.10.33 works great. Thanks a lot for the quick fix!
